---
title: "R Final Project"
author: "Erica Faucher"
format: pdf
editor: visual
---

**A comparison of bulk stool and FTA card stool sampling methods in terms of bacterial diversity**

[Abstract]{.underline}

This project examines the human gut microbiome, which is influenced by a person's diet, delivery mode, geography, and food supply, to name a few. FTA cards and bulk stool samples are two methods used to analyze the bacterial make up of the microbiome; however, it has not been tested whether FTA cards have an effect on bacterial diversity in samples. The stool samples of seven participants were taken using both methods, and then sequenced using 16S rRNA gene sequencing. The data was analyzed in R in terms of bacterial diversity by phyla. Multiple graphs were created to show the different sampling methods' bacterial makeup, as well as statistical results. It was found that there is a statistical difference between the FTA cards and bulk stool sample methods in terms of bacterial diversity. 



[Introduction]{.underline}

The human gut microbiome contains a multitude of different microbes, especially bacteria. The diversity of gut bacteria varies from person to person, and is affected by many factors, such as delivery mode (whether a person was born via cesarian section or vaginal birth), infant feeding method (breast milk or formula), diet, geography, food supply, stress, and antibiotics (Cresci and Bawden, 2016). Stool samples are typically used for analyzing the gut microbiome with gene sequencing. Stool samples are typically collected with the bulk stool sample method, which requires immediate freezing. However, this method is not feasible for hard-to-reach populations, such as people who live in more rural areas. Bulk stool also cannot be mailed, since the samples must be processed within 24 hours, or as soon as possible. Fast technology for analysis of nucleic acids (FTA) cards are a newer and alternative method. A chemical on the cards fixes the bacteria and allows them to be stored at room temperature. FTA cards have the potential to make stool sample collection easier for researchers to process, as well as allow for more gut microbiome research to be done in harder to reach populations.

The efficacy of FTA cards against bulk stool sample collection, in terms of bacterial diversity and richness, has not been conducted. Bacterial diversity in the gut is important because of its relationship with one's overall health. Lower bacterial diversity in the gut microbiome, as well as an increase of pro-inflammatory bacteria, is associated with Alzheimer's disease. The gut microbiome may have a role in the regulating neuroinflammatory disorders, such as multiple sclerosis and Parkinson's disease. Changes to the gut microbiome can affect the progression of illnesses in the body, as well as neurotransmitters like serotonin (Menezes and Shah, 2024).

This project focuses on analyzing seven stool samples that were taken with both the bulk stool and FTA card sampling methods. It was predicted that there will be no difference in bacterial diversity between the two stool sampling methods.

[Methods]{.underline}

The gold standard bulk stool sample collection method was compared to the FTA card collection method, and the microbial differences between the two methods was analyzed. In the summer of 2023, seven volunteers donated their stool samples, which were collected with both the bulk stool and FTA card sample methods. The bulk stool samples were homogenized, made into three aliquots, and stored at -80ºC within 24 hours of collection. The person who produced the stool sample smeared a small amount of the sample into the four circles on the FTA card, which immediately fixed the sample and allowed it to be stable at room temperature for a long period of time. 16S rRNA gene sequencing was then conducted on all the samples.

The data was transferred into an Excel spreadsheet, which was uploaded into R. The data was then relabeled, and the taxonomy column was split so that there was one column for each taxonomic level. Upon creating graphs of the data, it was determined that the data set was quite large, and that a smaller subset of the data would be analyzed in the context of this paper. Thus, only the phyla of bacteria in the samples was compared.

One bar graph was created for the average bacteria phyla in all of the bulk samples, and another bar graph for the average number of bacteria in all the FTA card samples. A third bar graph was created for the average bacteria in all samples, FTA and bulk. A Shannon Diversity Index was run to examine the difference in diversity for the two sample methods for each phylum. A second Shannon test was done to compare the diversity between the sampling methods, and a box plot was produced to visualize the results. A Wilcox test was then run to test the statistical significance of the diversity differences between the two methods. The code for all of these graphs and analyses can be located on Git Hub at https://github.com/emf2003/R_Final_Project.git. 

[Results]{.underline}


```{r, echo=FALSE, message=FALSE, warning=FALSE, error=FALSE}

# Load necessary libraries
library(readxl)   # for reading Excel files
library(dplyr)    # for data manipulation
library(stringr)  # for string manipulation
library(splitstackshape)

R_Final_Project_Data <- read_excel("R_Final_Project_Data.xlsx")

Split_Final <- cSplit(R_Final_Project_Data, "taxonomy", sep = "; ", direction = "wide")

#Rename columns
Split_Final <- Split_Final %>%
  rename(Domain = taxonomy_1, 
         Phylum = taxonomy_2, 
         Class = taxonomy_3, 
         Order = taxonomy_4, 
         Family = taxonomy_5, 
         Genus = taxonomy_6, 
         Species = taxonomy_7)

```

```{r, echo=FALSE, message=FALSE, warning=FALSE, error=FALSE}
#average Phyla bacteria of each Participant in their FTA and Bulk

Averages_All <- Split_Final %>%
  mutate(P85_mean_Bulk = rowMeans(select(., `P85-BULK1`, `P85-BULK2`, `P85-BULK3`), na.rm = TRUE)) %>%
  mutate(P85_mean_FTA = rowMeans(select(., `P85-FTA1`, `P85-FTA2`, `P85-FTA3`), na.rm = TRUE)) %>%
  mutate(P86_mean_Bulk = rowMeans(select(., `P86-BULK1`, `P86-BULK2`, `P86-BULK3`), na.rm = TRUE)) %>%
  mutate(P86_mean_FTA = rowMeans(select(., `P86-FTA1`, `P86-FTA2`, `P86-FTA3`), na.rm = TRUE)) %>%
  mutate(P87_mean_Bulk = rowMeans(select(., `P87-BULK1`, `P87-BULK2`, `P87-BULK3`), na.rm = TRUE)) %>%
  mutate(P87_mean_FTA = rowMeans(select(., `P87-FTA1`, `P87-FTA2`, `P87-FTA3`), na.rm = TRUE)) %>%
  mutate(P88_mean_Bulk = rowMeans(select(., `P88-BULK1`, `P88-BULK2`, `P88-BULK3`), na.rm = TRUE)) %>%
  mutate(P88_mean_FTA = rowMeans(select(., `P88-FTA1`, `P88-FTA2`, `P88-FTA3`), na.rm = TRUE)) %>%
  mutate(P89_mean_Bulk = rowMeans(select(., `P89-BULK1`, `P89-BULK2`, `P89-BULK3`), na.rm = TRUE)) %>%
  mutate(P89_mean_FTA = rowMeans(select(., `P89-FTA1`, `P89-FTA2`, `P89-FTA3`), na.rm = TRUE)) %>%
  mutate(P90_mean_Bulk = rowMeans(select(., `P90-BULK1`, `P90-BULK2`, `P90-BULK3`), na.rm = TRUE)) %>%
  mutate(P90_mean_FTA = rowMeans(select(., `P90-FTA1`, `P90-FTA2`, `P90-FTA3`), na.rm = TRUE)) %>%
  mutate(P91_mean_Bulk = rowMeans(select(., `P91-BULK1`, `P91-BULK2`, `P91-BULK3`), na.rm = TRUE)) %>%
  mutate(P91_mean_FTA = rowMeans(select(., `P91-FTA1`, `P91-FTA2`, `P91-FTA3`), na.rm = TRUE))

Averages_Only <- Averages_All %>%
  select("Domain", "Phylum","Class" ,"Order", "Family",       
"Genus", "Species", "P85_mean_Bulk", "P85_mean_FTA" , "P86_mean_Bulk", "P86_mean_FTA" ,
"P87_mean_Bulk", "P87_mean_FTA"  ,"P88_mean_Bulk", "P88_mean_FTA" , "P89_mean_Bulk", "P89_mean_FTA" ,
"P90_mean_Bulk", "P90_mean_FTA" , "P91_mean_Bulk", "P91_mean_FTA" )

```

```{r, echo=FALSE, message=FALSE, warning=FALSE, error=FALSE}

scale_to_range <- function(x, a, b) {
    a + ((x - min(x, na.rm = TRUE)) * (b - a)) / (max(x, na.rm = TRUE) - min(x, na.rm = TRUE))
}

normalized_data_av <- as.data.frame(lapply(Averages_Only, function(col) {
    if (is.numeric(col)) {
        scale_to_range(col, 0, 1000)  # Example: Scale to range [0, 10]
    } else {
        col
    }
}))

```

Figure 1 (below) depicts a bar graph of the average number of bacteria in each phyla for all seven participants' bulk stool samples. The phyla with the highest averages across all samples include Firmicutes and Bacteroidota. Sample 1 has the highest average of bacteria in the Actinobacteriota and Verrucomicrobiota phyla. Sample 1 also has the lowest average of Bacteroidota bacteria. Sample 6 has the lowest average number of bacteria in the Firmicutes phylum.

```{r, fig.cap = "The average number of bacteria in each phyla in all bulk stool samples.", echo=FALSE, message=FALSE, warning=FALSE, error=FALSE}

library(ggplot2)
library(dplyr)

normalized_data_av <- normalized_data_av 

# Create a long-format data frame manually for plotting
Averages_Norm_long <- data.frame(
  Phylum = rep(normalized_data_av$Phylum, times = 7),  
  Measurement = rep(c("P85_mean_Bulk", "P86_mean_Bulk", "P87_mean_Bulk", 
                      "P88_mean_Bulk", "P89_mean_Bulk", "P90_mean_Bulk", "P91_mean_Bulk"), 
                    each = nrow(normalized_data_av)),
  Value = c(normalized_data_av$P85_mean_Bulk, normalized_data_av$P86_mean_Bulk, normalized_data_av$P87_mean_Bulk, 
            normalized_data_av$P88_mean_Bulk, normalized_data_av$P89_mean_Bulk, normalized_data_av$P90_mean_Bulk, 
            normalized_data_av$P91_mean_Bulk)
)

custom_labels <- c(
  "P85_mean_Bulk" = "1",
  "P86_mean_Bulk" = "2",
  "P87_mean_Bulk" = "3",
  "P88_mean_Bulk" = "4",
  "P89_mean_Bulk" = "5",
  "P90_mean_Bulk" = "6",
  "P91_mean_Bulk" = "7"
)


ggplot(Averages_Norm_long, aes(x = Measurement, y = Value, fill = Phylum)) +
  geom_bar(stat = "identity", position = "dodge") +  # Position bars side by side
  scale_x_discrete(labels = custom_labels) +
  scale_fill_discrete(labels = function(x) gsub("p_|_", "", x)) +
  labs(x = "Bulk Sample #", y = "Average Number of Bacteria", title = "Bacteria Phyla in Bulk Samples")

```

Figure 2 (below) shows the average number of bacteria in each phyla for the FTA card samples across all participants. The phyla with the highest averages across all samples were Firmicutes and Bacteroidota. Sample 1 had the highest average number of Actinobacteriota , Verrucomicrobiota, and Desulfobacterota, but the lowest average number of Bacteroidota. Sample 6 had the lowest average number of bacteria in the Firmicutes phylum.

```{r, fig.cap = "Average number of bacteria, by phyla, in all FTA card stool samples.", echo=FALSE, message=FALSE, warning=FALSE, error=FALSE}
library(ggplot2)
library(dplyr)

# Create a long-format data frame manually for plotting
Averages_Norm_long <- data.frame(
  Phylum = rep(normalized_data_av$Phylum, times = 7),  # Repeat Phylum for each measurement
  Measurement = rep(c("P85_mean_FTA", "P86_mean_FTA", "P87_mean_FTA", 
                      "P88_mean_FTA", "P89_mean_FTA", "P90_mean_FTA", "P91_mean_FTA"), 
                    each = nrow(normalized_data_av)),
  Value = c(normalized_data_av$P85_mean_FTA, normalized_data_av$P86_mean_FTA, normalized_data_av$P87_mean_FTA, 
            normalized_data_av$P88_mean_FTA, normalized_data_av$P89_mean_FTA, normalized_data_av$P90_mean_FTA, 
            normalized_data_av$P91_mean_FTA)
)


custom_labels <- c(
  "P85_mean_FTA" = "1",
  "P86_mean_FTA" = "2",
  "P87_mean_FTA" = "3",
  "P88_mean_FTA" = "4",
  "P89_mean_FTA" = "5",
  "P90_mean_FTA" = "6",
  "P91_mean_FTA" = "7"
)

# Create the plot
ggplot(Averages_Norm_long, aes(x = Measurement, y = Value, fill = Phylum)) +
  geom_bar(stat = "identity", position = "dodge") +  # Position bars side by side
  scale_x_discrete(labels = custom_labels) +
  scale_fill_discrete(labels = function(x) gsub("p_|_", "", x)) +
  labs(x = " FTA Sample #", y = "Average Number of Bacteria", title = "Bacteria Phyla in FTA Card Samples")

```

Figure 3 (below) shows the average number of bacteria in the bulk and FTA samples for all participants. The average number of bacteria in the bulk samples was higher than the average number of bacteria in the FTA card stool samples, across all samples. The FTA cards for samples 1, 2, and 3 have the lowest average numbers of bacteria. FTA sample 6 has the highest average number of bacteria compared to the other FTA card samples. Bulk sample 2 has the highest average number of bacteria overall.

```{r, fig.cap = "Average number of bacteria in FTA and bulk stool samples for all participants.", echo=FALSE, message=FALSE, warning=FALSE, error=FALSE}

#Note: data is not normalized here, on purpose. 

library(ggplot2)
library(dplyr)

# Create a long-format data frame manually for plotting
Averages_Only_long2 <- data.frame(
  Domain = rep(Averages_Only$Domain, times = 7),  # Repeat Phylum for each measurement
  Measurement = rep(c("P85_mean_FTA", "P86_mean_FTA", "P87_mean_FTA", 
                      "P88_mean_FTA", "P89_mean_FTA", "P90_mean_FTA", "P91_mean_FTA","P85_mean_Bulk", "P86_mean_Bulk", "P87_mean_Bulk", "P88_mean_Bulk", "P89_mean_Bulk", "P90_mean_Bulk", "P91_mean_Bulk"), 
                    each = nrow(Averages_Only)),
  Value = c(Averages_Only$P85_mean_FTA, Averages_Only$P86_mean_FTA, Averages_Only$P87_mean_FTA, 
            Averages_Only$P88_mean_FTA, Averages_Only$P89_mean_FTA, Averages_Only$P90_mean_FTA, 
            Averages_Only$P91_mean_FTA,Averages_Only$P85_mean_Bulk, Averages_Only$P86_mean_Bulk,     Averages_Only$P87_mean_Bulk, Averages_Only$P88_mean_Bulk, Averages_Only$P89_mean_Bulk, Averages_Only$P90_mean_Bulk, 
Averages_Only$P91_mean_Bulk)
)

custom_labels <- c(
  "P85_mean_Bulk" = "Bulk 1",
  "P86_mean_Bulk" = "Bulk 2",
  "P87_mean_Bulk" = "Bulk 3",
  "P88_mean_Bulk" = "Bulk 4",
  "P89_mean_Bulk" = "Bulk 5",
  "P90_mean_Bulk" = "Bulk 6",
  "P91_mean_Bulk" = "Bulk 7",
  "P85_mean_FTA" = "FTA 1",
  "P86_mean_FTA" = "FTA 2",
  "P87_mean_FTA" = "FTA 3",
  "P88_mean_FTA" = "FTA 4",
  "P89_mean_FTA" = "FTA 5",
  "P90_mean_FTA" = "FTA 6",
  "P91_mean_FTA" = "FTA 7"
)

# Create the plot
ggplot(Averages_Only_long2, aes(x = Measurement, y = Value, fill = Measurement)) +
  geom_bar(stat = "identity", position = "dodge") +
  scale_fill_manual(values = c(
    "P85_mean_Bulk" = 6,
    "P86_mean_Bulk" = 6,
    "P87_mean_Bulk" = 6,
    "P88_mean_Bulk" = 6,
    "P89_mean_Bulk" = 6,
    "P90_mean_Bulk" = 6,
    "P91_mean_Bulk" = 6,
    "P85_mean_FTA" = 5,
    "P86_mean_FTA" = 5,
    "P87_mean_FTA" = 5,
    "P88_mean_FTA" = 5,
    "P89_mean_FTA" = 5,
    "P90_mean_FTA" = 5,
    "P91_mean_FTA" = 5)) +
  scale_x_discrete(labels = custom_labels) +  # Ensure custom_labels is defined
  labs(x = "Sample", y = "Average Number of Bacteria", title = "Bacteria in All Samples") +
  theme(axis.text.x = element_text(angle = 45, hjust = 1)) +
  theme(legend.position = "none")

```

A Shannon Diversity Index was run and was organized by phylum. Phyla Patescibacteria and Cyanobacteria had the lowest values compared to the other phyla. The bulk method shows higher values than the FTA values for all phyla except Patescibacteria and Verrucomicrobiota.

```{r, echo=FALSE, message=FALSE, warning=FALSE, error=FALSE}
library(vegan)

shannon_data <- Split_Final %>%
  na.omit() %>%
  select(Phylum, `P85-FTA1`, `P85-FTA2`, `P85-FTA3`, 
         `P86-FTA1`, `P86-FTA2`, `P86-FTA3`, 
         `P87-FTA1`, `P87-FTA2`, `P87-FTA3`, 
         `P88-FTA1`, `P88-FTA2`, `P88-FTA3`, 
         `P89-FTA1`, `P89-FTA2`, `P89-FTA3`, 
         `P90-FTA1`, `P90-FTA2`, `P90-FTA3`, 
         `P91-FTA1`, `P91-FTA2`, `P91-FTA3`)

shannon_by_phylum <- shannon_data %>%
  group_by(Phylum) %>%
  summarize(across(everything(), sum, .names = "total_{.col}")) %>%
  ungroup()

shannon_results <- shannon_by_phylum %>%
  rowwise() %>%
  mutate(Shannon = diversity(c_across(starts_with("total_")), 
                             index = "shannon")) %>%
  select(Phylum, Shannon)

```

```{r, fig.cap = "Shannon Diversity Index by Phylum for both sampling methods.", echo=FALSE, message=FALSE, warning=FALSE, error=FALSE, results='hide'}
#put graphs together

library(dplyr)
library(vegan)
library(ggplot2)

# Step 1: Prepare the data
shannon_dataBulk <- Split_Final %>%
  na.omit() %>%
  select(Phylum, `P85-BULK1`, `P85-BULK2`, `P85-BULK3`,`P86-BULK1`, `P86-BULK2`, `P86-BULK3`, `P87-BULK1`, `P87-BULK2`, `P87-BULK3`, `P88-BULK1`, `P88-BULK2`, `P88-BULK3`, `P89-BULK1`, `P89-BULK2`, `P89-BULK3`,`P90-BULK1`, `P90-BULK2`, `P90-BULK3`, `P91-BULK1`, `P91-BULK2`, `P91-BULK3`)

# Step 2: Group data by Phylum and sum sample values
shannon_by_phylumB <- shannon_dataBulk %>%
  group_by(Phylum) %>%
  summarize(across(everything(), sum, .names = "total_{.col}")) %>%
  ungroup()

# Step 3: Calculate the Shannon diversity index for each Phylum
shannon_resultsB <- shannon_by_phylumB %>%
  rowwise() %>%
  mutate(Shannon = diversity(c_across(starts_with("total_")), 
                             index = "shannon")) %>%
  select(Phylum, Shannon)


shannon_resultsFTA <- shannon_results %>%
  rename(ShannonFTA = Shannon)

shannon_resultsB %>% rename(Shannon_Bulk = Shannon)


shannon_all <- left_join(shannon_resultsB, shannon_resultsFTA, by = join_by(Phylum == Phylum))

shannon_all <- shannon_all %>%
  rename(Shannon_Bulk = Shannon, Shannon_FTA = ShannonFTA)

library(knitr)
library(kableExtra)
library(tidyverse)
shannon_all_long <- shannon_all %>%
  pivot_longer(names_to = 'method', values_to = 'Shannon_Both', cols = Shannon_Bulk:Shannon_FTA)


ggplot(shannon_all_long, aes(x = Phylum, y = Shannon_Both, fill = method)) +
  geom_bar(stat = "identity", position = "dodge") + 
  theme_minimal() +
  labs(title = "Shannon Diversity Index by Phylum", 
       x = "Phylum", 
       y = "Shannon Diversity") +
  theme(axis.text.x = element_text(angle = 45, hjust = 1)) +
  scale_x_discrete(labels = function(x) gsub("p_|_", "", x)) +
  guides(fill = guide_legend(title = "Method")) +
  scale_color_manual(labels = c("Bulk", "FTA")) 

```

Figure 5 (below) shows a box plot of the results of the second Shannon Diversity Index calculated on the diversity between the two sampling methods. The bulk sampling data has one outlier, shown in the graph as a black dot. The bulk method has a higher median value than the FTA card sampling method. The FTA card method has a larger interquartile range than the bulk sampling method. 

```{r, fig.cap = "Box plot depicting the Shannon Diversity Index to compare the bacterial diversity for both methods." , echo=FALSE, message=FALSE, warning=FALSE, error=FALSE, results='hide'}
library(ggplot2)
library(dplyr)

# Subset FTA and Bulk samples
FTA_data <- Split_Final %>%
  select(starts_with("P") & ends_with("FTA1") | ends_with("FTA2") | ends_with("FTA3"))

Bulk_data <- Split_Final %>%
  select(starts_with("P") & ends_with("BULK1") | ends_with("BULK2") | ends_with("BULK3"))

# Transpose data
FTA_data_t <- as.data.frame(t(FTA_data))
Bulk_data_t <- as.data.frame(t(Bulk_data))

# Calculate Shannon diversity
shannon_FTA <- diversity(FTA_data_t, index = "shannon")
shannon_Bulk <- diversity(Bulk_data_t, index = "shannon")

# Create data frame for plotting
shannon_combined <- data.frame(
  Method = rep(c("FTA", "Bulk"), times = c(length(shannon_FTA), length(shannon_Bulk))),
  Shannon = c(shannon_FTA, shannon_Bulk)
)

# Plot the data
ggplot(shannon_combined, aes(x = Method, y = Shannon, fill = Method)) +
  geom_boxplot() +
  theme_minimal() +
  labs(title = "Shannon Diversity by Method", 
       x = "Method", 
       y = "Shannon Diversity") +
  theme(legend.position = "none")
  

```

The p-value for the Wilcox test was 0.03359. 

```{r, echo=FALSE, message=FALSE, warning=FALSE, error=FALSE, results='hide'}

str(shannon_combined)

shannon_combined$Method <- as.factor(shannon_combined$Method)

wilcox.test(Shannon ~ Method, data = shannon_combined)

```

[Discussion]{.underline}

This project focused on examining the diversity of phyla between the two methods of stool sampling, FTA card and bulk stool. Figures 1 and 2 show that the most common phyla that make up the gut microbiome are Firmicutes and Bacteroidota. However, the FTA samples had higher averages of bacteria belonging to the phyla Verrucomicrobiota and Proteobacteria than in the bulk stool samples. The bulk samples tended to have higher averages of Actinobacteriota than the FTA card samples. As shown in Figure 3, The average number of bacteria in the bulk samples was higher across all participants in the study, indicating that stool sample collection yields a greater number of bacteria than in FTA card sample collection.

The first Shannon Diversity Index (Figure 4) results show that there was overall more diversity in the bulk stool samples than the FTA card samples across all phyla (except for Patescibacteria and Verrucomicrobiota). The second Shannon Diversity Index was run to compare the bacterial diversity between the two sampling methods. The box plot in Figure 5 (the results of the second Shannon, comparing the methods) shows that the bulk sample method has a higher median than the FTA card method, indicating that the bulk sample method has a higher bacterial diversity. The p-value for the Wilcox test was 0.03359, which is less than 0.05, meaning that there is a statistically significant difference between the two sampling methods. There is a difference in the diversity of phyla between the FTA and bulk stool sampling methods. This goes against the hypothesis, that there would be no difference in diversity between the sampling methods. This means that researchers must take into account this difference when choosing whether to gather stool samples using the FTA card or bulk stool sampling method.

It is important to note that this study has a small sample size (7), which may have an impact on the results. However, smaller sample sizes for studies is not unheard of. A study on sampling methods measuring HIV viral load had 51 participants (Jaumdally *et al,* 2017), and a study on pharmacokinetic properies of insulin icodec had a sample size of 46 (Pieber *et al,* 2023). One study on the effect of *Bifidobacterium breve* M-16V and mood in humans had a sample size of just 30 participants (Mutoh *et al,* 2024).

Another factor that can impact the data is that the participants are aliquoting their samples themselves on FTA cards. It is up to the person who produces the stool sample to accurately swab the sample into the circles on the card. If done incorrectly (ie not swabbing enough sample, or swabbing too much/leaving large pieces of stool), this can affect the number of bacteria in the sample. This can also affect the 16S results. In a bulk stool sample, the participant is still responsible for producing a clean sample (ie a sample not contaminated with urine), but is not responsible for aliquoting the sample.





[References]{.underline}

Cresci, G. A., & Bawden, E. (2015). The Gut Microbiome: What we do and don’t know. Nutrition in Clinical Practice : Official Publication of the American Society for Parenteral and Enteral Nutrition, 30(6), 734. https://doi.org/10.1177/0884533615609899

Jaumdally, S. Z., Jones, H. E., Hoover, D. R., Gamieldien, H., Kriek, J.-M., Langwenya, N., Myer, L., Passmore, J.-A. S., & Todd, C. S. (2017). Comparison of sampling methods to measure HIV RNA viral load in female genital tract secretions. American Journal of Reproductive Immunology (New York, N.Y.: 1989), 77(3). https://doi.org/10.1111/aji.12619

Menezes, A. A., & Shah, Z. A. (2024). A Review of the Consequences of Gut Microbiota in Neurodegenerative Disorders and Aging. Brain Sciences, 14(12), Article 12. https://doi.org/10.3390/brainsci14121224

Mutoh, N., Moriya, M., Xu, C., Kato, K., Arai, S., Iwabuchi, N., Tanaka, M., Jinghua, Y., Itamura, R., Sakatani, K., & Warisawa, S. (2024). Bifidobacterium breve M-16V regulates the autonomic nervous system via the intestinal environment: A double-blind, placebo-controlled study. Behavioural Brain Research, 460, 114820. https://doi.org/10.1016/j.bbr.2023.114820

Pieber, T. R., Asong, M., Fluhr, G., Höller, V., Kristensen, N. R., Larsen, J. H., Ribel-Madsen, R., Svehlikova, E., Vinther, S., Voortman, M., & Haahr, H. (2023). Pharmacokinetic and pharmacodynamic properties of once-weekly insulin icodec in individuals with type 2 diabetes. Diabetes, Obesity & Metabolism, 25(12), 3716–3723. https://doi.org/10.1111/dom.15266

R Core Team (2024). _R: A Language and Environment for Statistical Computing_. R Foundation for
  Statistical Computing, Vienna, Austria. <https://www.R-project.org/>.




